web/packages/teleport/src/Discover/useDiscover.tsx

```diff
@@ -100,27 +126,86 @@ export function DiscoverProvider<T = any>(
     return addIndexToViews(selectedResource.views);
   }, [selectedResource.views, resourceState]);
 
+  useEffect(() => {
+    const emitAbortOrSuccessEvent = () => {
+      if (ref.current.currEventName === DiscoverEvent.Completed) {
+        emitEvent({ stepStatus: DiscoverEventStatus.Success });
```

Comment:
i didn't think it would make sense to send a `abort` signal on the completed step (it's just letting user know they finished), but i can make it so that if user `does not click` on the two button we provided (as shown in picture), i can send an abort signal instead

![image](https://user-images.githubusercontent.com/43280172/214180903-e3646b75-ce7c-401c-adef-81f47beb1d89.png)

Reply:
Looks good to me :+1: